docs/development/release_management.rst

```diff
@@ -38,6 +38,11 @@ Pre-Release
    Keep this issue updated as you proceed through the release process for
    transparency.
 
+#. Copy a link of the latest release or release candidate from the `Tails apt repo
+   <https://deb.tails.boum.org/dists/>`_ and include it in the issue. This is to ensure QA tests
+   against the latest release/ release candidate and can report any release candidate bugs early to
+   Tails.
```

Comment:
Because that repo can be a bit hard to navigate, it may be useful to add a link to https://tails.boum.org/contribute/calendar/ as well ("compare with the the [Tails release calendar](https://tails.boum.org/contribute/calendar/)").